staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -100,6 +105,10 @@ func ValidateCustomResourceDefinitionSpec(spec *apiextensions.CustomResourceDefi
 
 	allErrs = append(allErrs, ValidateCustomResourceDefinitionNames(&spec.Names, fldPath.Child("names"))...)
 
+	if utilfeature.DefaultFeatureGate.Enabled(apiextensionsfeatures.CustomResourceValidation) {
```

Comment:
@deads2k Had done that earlier. But after this discussion: https://github.com/kubernetes/community/pull/913#discussion_r133227732, it was decided to drop the feature: https://github.com/kubernetes/kubernetes/pull/47263/files#diff-18bbf71961e0d43572fc7357183260feR51